modules/db/src/main/scala/gem/dao/StaticConfigDao.scala

```diff
@@ -116,21 +116,19 @@ object StaticConfigDao {
     /** F2 Statements. */
     object F2 {
 
-      import Instrument.Flamingos2
-
       def select(sid: Int): Query0[StaticConfig.F2] =
         sql"""
           SELECT mos_preimaging
             FROM static_f2
-           WHERE static_id = $sid AND instrument = ${Flamingos2: Instrument}
+           WHERE static_id = $sid
```

Comment:
Why this change?

Reply:
While writing the same code for GNIRS we realized there is already postgres constraint for the instrument (`is_f2`), so the extra `AND` is not needed.